src/components/RichTextEditor/serializer/serializeToHtml.ts

```diff
@@ -46,14 +46,9 @@ export const serializeNodesToHtml = (
 
     let html = '';
     for (let i = 0, len = nodes.length; i < len; i++) {
-        const node = nodes[i];
-        if (isEmptyNode(node)) {
-            html += '<br />';
```

Comment:
Is it fine to remove this? I figured you might need to add the empty character here as well?

Reply:
The problem with the solution we had in place was that you are gonna loose information of empty lines, as they were just `<br/>`, but the line-height of the selected text-style was removed as it didn't know about that. So now we are parsing all empty lines as correct leafs, which will end up in correct spacing.